src/rvpt/rvpt.cpp

```diff
@@ -182,10 +183,24 @@ void RVPT::update_imgui()
         ImGui::PushItemWidth(80);
         ImGui::SliderInt("AA", &render_settings.aa, 1, 64);
         ImGui::SliderInt("Max Bounce", &render_settings.max_bounces, 1, 64);
```

Comment:
Would it be possible to put these both on the same line?
Reason being; These two are somewhat related, so maybe add some type of structure to them?

Reply:
I don't think there's room in the default size.  I'd argue it's fine as is.